pwiz_tools/Skyline/EditUI/FindNodeDlg.cs

```diff
@@ -87,6 +87,15 @@ public bool CaseSensitive
             set { cbCaseSensitive.Checked = value; }
         }
 
+        protected override void OnShown(EventArgs e)
+        {
+            if (!Program.SkylineOffscreen)
+            {
+                this.Focus(); // It's natural to want to start typing the search term immediately after hitting ctrl-F

```

Comment:
I am not sure why this would be necessary.
Whenever I bring up the Find dialog in Skyline, it always has the focus and I can begin typing immediately.
Does the Find dialog behave differently on your computer?

Reply:
Yes, for whatever reason this seemed to make for a more consistent experience - I often find I need to click before I can type in the search term. Maybe it's because of something else going on (offscreen tests?) that a normal user wouldn't see. Happy to back it out if it feels weird.